docs/barbican-kms-plugin/using-barbican-kms-plugin.md

```diff
@@ -60,7 +60,7 @@ $ openstack secret order get http://hostname:9311/v1/orders/e477a578-4a46-4c3f-b
 ```
 
 
-### Add the Key ID in your cloud-config file
+### Add the key ID in your cloud-config file
```

Comment:
> hm, why did you change the `s/Key/key/` only here?

Because I thought these other keys are some inbuilt variables that should not change and the one I changed is in the comment part so I thought to change. Should I update the others as well?